From: Gunter Königsmann <wxmaxima@physikbuch.de>
Description: Removed the old menu file that isn't really used any more.
Author: Gunter Königsmann <wxmaxima@physikbuch.de>
Forwarded: Yes
--- a/configure.ac
+++ b/configure.ac
@@ -250,7 +250,6 @@
         wxmaxima.spec
         data/Info.plist
 	data/wxMaxima.desktop
-	data/wxmaxima.menu
 ])
 
 AC_OUTPUT
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -12,10 +12,6 @@
 appicondir = $(datarootdir)/pixmaps
 dist_appicon_DATA = wxmaxima.svg wxmaxima.png text-x-wxmaxima-batch.svg text-x-wxmathml.svg wxmaxima-16.xpm wxmaxima-32.xpm
 
-menudir = $(datarootdir)/menu
-dist_menu_DATA = wxmaxima.menu
-
-
 
 mandatadir = $(datadir)/man/man1
 dist_mandata_DATA = wxmaxima.1
--- a/data/wxmaxima.menu
+++ /dev/null
@@ -1,6 +0,0 @@
-?package(wxmaxima):\ needs="X11"\
-    section="Applications/Science/Mathematics"\ title="wxMaxima"\
-    longtitle="GUI for the computer algebra system Maxima"\
-    icon16x16="/usr/local/share/pixmaps/wxmaxima-16.xpm"\
-    icon32x32="/usr/local/share/pixmaps/wxmaxima-32.xpm"\
-    command="/usr/bin/wxmaxima"
